ogr/abstract.py

```diff
@@ -213,21 +213,27 @@ def __str__(self) -> str:
 class Release:
     def __init__(
         self,
-        title: str,
-        body: str,
         tag_name: str,
         url: str,
         created_at: str,
         tarball_url: str,
         git_tag: GitTag,
+        project: Optional["GitProject"],
```

Comment:
```suggestion
        project: "GitProject",
```

It is not needed now...